korman/nodes/node_avatar.py

```diff
@@ -15,7 +15,8 @@
 
 import bpy
 from bpy.props import *
-from collections import OrderedDict
+from __future__ import annotations
```

Comment:
Any time that you do a `from __future__` import, it needs to be the first import statement. This, unfortunately, errors out. You'll need to move it up to the very top of the list of imports.

Reply:
Done!